README.md

```diff
@@ -144,7 +82,8 @@ Next, we inform BayBE about the available "control knobs", that is, the underlyi
 system parameters we can tune to optimize our targets. This also involves specifying 
 their values/ranges and other parameter-specific details.
 
-For our example, we assume that we can control the following three quantities:
+For our example, we assume that we can control three quantities – `Granularity`,

```

Comment:
I think I put that here but `quantity` is not a good word for `type of solvent` or `granularity`
suggest `parameters`

Reply:
fixed